widgetssdk/src/main/java/com/glia/widgets/core/fileupload/FileAttachmentRepository.java

```diff
@@ -18,23 +24,17 @@
 import java.util.stream.Stream;
 
 public class FileAttachmentRepository {
```

Comment:
Just out of interest, what  is the reason behind moving functions from the previous place to another? I'm asking because with changed order, it's very difficult to review exact changes and understand what if anything was  changed  or just moved to another place.